### DISCUSSION TITLE
How to turn off LBB

### DISCUSSION THREAD

**Discussion by roshin8** [outdoorbits/little-backup-box#238] [Q&A] [answered]
Once i install it, and the LBB hotspot is running, how do i revert/turn off everything and then get connected back to my exisiting WiFi? 
Also how do i prevent LBB services from running in the background if any?

**Comment by outdoorbits** [comment]
I'm not sure if I got your questions correctly. You want to shutdown lbb? You can do this by web UI. Can you access you web UI?
What do you mean when you write "get connected back to my exisiting WiFi"? What do you want to connect, your lbb?
What do you mean when you write "prevent LBB services from running in the background"? Usually it is active when RPi is on.

**Comment by roshin8** [comment]
Sorry i was installing LBB on rapberrypi GUI version and not Lite.
But after installing LBB on Lite version, i do see the LBB hotspot but after connecting to the LBB wifi hotspot and selecting my existing wifi from the captive portal, the LBB wifi gets disconnected and i never see the LBB wifi hotspot again. 
Even after rebooting, i don't see the LBB hotspot

**Comment by outdoorbits** [comment]
You have one WiFi adapter only and this connects to your WiFi. If there is no known WiFi, the hotspot comes up again. 
Else you can configure LBB to send you an email containing the link when connected to the WiFi. And you can use a display. 
Or you just add a WiFi USB dongle and you can have both.

**Reply by roshin8** [reply]
Okay I was able to access the Web UI of LBB by going to the IP address of the box. 
Is it possible to re-configure comitup to connect to different Wifi? i.e get back the LBB hotspot running where I choose i go to the captive portal and choose a wifi

**Reply by roshin8** [reply]
Also it says that I can 

> alternatively can reach the web UI of Little Backup Box when you are connected to its hotspot as follows:
> https://10.41.0.1 (secure, certificate can't be verified automatically, please confirm it)
> http://10.41.0.1:8000 (insecure) 

But going to these IP addresses don't take me to the web UI of LBB

**Reply by roshin8** [reply]
Also, do you have tutorial on how to set up LBB to send you email containing the link?

**Comment by outdoorbits** [comment]
It's just using the settings page of the UI. What question do you have exactly?

**Reply by roshin8** [reply]
How to connect to a different wifi? i.e get back the LBB hotspot running again?

**Comment by outdoorbits** [comment]
This is the IP when you are connected to the hotspot of LBB. When connected to the WiFi, you need another.

**Comment by outdoorbits** [comment]
Have a lot to the settings.

**Comment by outdoorbits** [comment, marked as answer]
In settings you will find the section COMITUP.

Message ID: 
***@***.***
